test/spec/features/grid/update_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe 'update grid' do
+  it 'validates log opts' do
+    k = run "kontena grid update --log-opt server=foo e2e"
+    expect(k.code).not_to eq(0)
+    expect(k.out).to match /Need to specify --log-driver when using --log-opt/
```

Comment:
added cli spec for this also. A bit redundant but didn't want to remove from e2e tests :D